build-system/utils.js

```diff
@@ -35,7 +35,6 @@ const {log, logWithoutTimestamp} = require('./log');
  * @param {string} cmd command to execute.
  */
 function timedExecOrDie(cmd) {
-  log('Running', cyan(cmd) + '...');
```

Comment:
Why remove this? I feel like this is really useful for reproducing CI failures locally

Reply:
Removed during CI because the full command is already printed at the start of the collapsible area. I've restored the statement for local development.